packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,22 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## January
 
+### Weekly release 2022.01.2
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.01.2](#bugs-fixed-in-2022-01-2)         | January 26, 2022           |
+
+#### Bugs fixed in 2022.01.2
+
+* When the [Create a new Binding](/docs/reference/api/roles/#create-a-new-binding) or the [Add more Members to a Binding](/docs/reference/api/roles/#add-more-members-to-a-binding) endpoints were called, and included all users or all groups in the request, the request didn't fail as expected. (OKTA-459994)
```

Comment:
Should we call these "endpoints"? There is just one endpoint, with different request types. Maybe use "Resource Set operations"?